cpp/client/ct.cc

```diff
@@ -753,7 +755,7 @@ static void DiagnoseCertChain() {
       continue;
     }
     LOG(INFO) << "SCT number " << i + 1 << ":\n" << sct.DebugString();
-    if (verifier != NULL) {
+    if (verifier.get()) {
```

Comment:
No, @benlaurie is just remembering "C from the late 70's", due to the CDC 6000 he keeps around to work on OpenSSL... :wink:

This was explicitly legislated as "proper C++" a long time ago (and proper C as well, from at least C89, I think?):

https://github.com/cplusplus/draft/blob/093c1eebd3e2a1682331e8b1bd16677cbdb79d6f/source/conversions.tex#L549